[Bug] TypeScript compilation fails due to incompatible import syntax in python-analyzer.ts

## Environment
- Node.js: v24.11.1
- npm: 11.6.2
- TypeScript: 5.8.3
- OS: macOS 26.1 (Darwin 25.1.0, ARM64)

## Problem
`npx tsc` fails with the following error:
```bash
src/engine/analyzer/python/common/python-analyzer.ts:1:8 - error TS1192: Module has no default export.

1 import SymAddress from '../../common/sym-address'
         ~~~~~~~~~~
```

## Reproduction Steps
1. Clone the repository
2. Run `npm install`
3. Run `npx tsc`

## Root Cause
Import/export style mismatch introduced in PR #50 (commit cc5de14d, 2025-11-14):
- `python-analyzer.ts:1` uses ES6 default import: `import SymAddress from '../../common/sym-address'`
- `sym-address.ts:164` uses CommonJS export: `module.exports = {...}`
- All other files in the project use `require()` to import `sym-address`:
  - `analyzer.ts:17`: `const SymAddress = require('./sym-address')`
  - `memSpace.ts:7`: `const symAddress = require('./sym-address')`
  - `callgraph-checker.ts:5`: `const symAddressCallgraph = require('../../engine/analyzer/common/sym-address')`

## Impact
- Blocks source code build for all users (since commit cc5de14d)

## Proposed Fix
Change `src/engine/analyzer/python/common/python-analyzer.ts:1` from:
```typescript
import SymAddress from '../../common/sym-address'
```
to:
```typescript
const SymAddress = require('../../common/sym-address')
```

## Related PR
This issue was introduced in PR #50 "fixbug: Differentiate VariableDeclaration and AssignmentExpression usage"
I'm willing to submit a PR to fix this regression if approved.

--- a/src/engine/analyzer/python/common/python-analyzer.ts
+++ b/src/engine/analyzer/python/common/python-analyzer.ts
@@ -1,4 +1,4 @@
-import SymAddress from '../../common/sym-address'
+const SymAddress = require('../../common/sym-address')
 
 const { Analyzer } = require('../../common')
 const CheckerManager = require('../../common/checker-manager')

